Assets/Scripts/Utils/DestroyOnPlay.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+namespace Utils
+{
+    public class DestroyOnPlay : MonoBehaviour
+    {
+        private void Awake()
+        {
+            Destroy(gameObject);
```

Comment:
Where do you use this?

Reply:
It's attached to the NotWalkableOceanPlane, which is used for generating the NavMesh

Reply:
But since the plane shouldn't be there in the actual game, I've added this little script

Reply:
I see